solidity/solidity.go

```diff
@@ -60,7 +54,7 @@ type SolidityProof struct {
 type Groth16CommitmentProof struct {
 	Proof         SolidityProof `json:"proof"`
 	Commitments   [2]*big.Int   `json:"commitments"`
-	CommitmentPok [2]*big.Int   `json:"commitment_pok"`
+	CommitmentPok [2]*big.Int   `json:"commitmentPok"`
```

Comment:
why this name change? (just wondering)

Reply:
nothing, just desperation with the internals of abi.ABI, to see if something changes..